src/ast/ddl.rs

```diff
@@ -3199,6 +3199,22 @@ pub struct CreateTrigger {
     ///
     /// [MsSql](https://learn.microsoft.com/en-us/sql/t-sql/statements/create-trigger-transact-sql?view=sql-server-ver16#arguments)
     pub or_alter: bool,
+    /// True if this is a temporary trigger, which is supported in SQLite.
```

Comment:
```suggestion
    /// True if this is a temporary trigger.
```

Reply:
Since it is solely supported in SQLite (that I know of), I thought it important to document it.